packages/vanilla-bundle/src/components/slick-vanilla-grid-bundle.ts

```diff
@@ -1434,7 +1434,7 @@ export class SlickVanillaGridBundle {
 
     if (this.slickGrid) {
       // find the new column reference pointer & re-assign the new editor to the internalColumnEditor
-      const columns = this.slickGrid.getColumns();
+      const columns = this._columnDefinitions;
```

Comment:
can you use the GETTER instead (without the `_` prefix) and there's no use in assigning a new variable from one to the other, just use it directly.
```ts
if (Array.isArray(this.columnDefinitions)) {
  const columnRef = this.columnDefinitions.find((col: Column) => col.id === column.id);
 //...
```

Reply:
You also have a lot of unit tests failing with this change, can you please fix them